php/util.php

```diff
@@ -393,6 +393,32 @@ function getUploadsPath( $user = null )
 	return( getProfilePath($user).'/torrents' );
 }
 
+function getWebUIJsonFile()
+{
+	global $profilePath;
+	$ret.= isset($profilePath) ? $profilePath : '../share';
+	$ret = str_replace('..', '', $ret);
+	
```

Comment:
Really? What we will have for "../../../blah-blah-blah", for example?

Reply:
Good catch, I will create the real path sooner. To resolve this problem.